server/src/catalog.rs

```diff
@@ -74,7 +79,6 @@ pub fn new(data_folder: &str) -> Catalog {
         Field::new("model", DataType::Binary, false),
         Field::new("gaps", DataType::Binary, false),
     ]));
-
```

Comment:
There should probably be a newline here.

Reply:
Fixed